c_scripts/network_throughput/net.c

```diff
@@ -7,12 +7,12 @@
 /*
 ** Source reference: https://github.com/EthanGYoung/gvisor_analysis/blob/master/experiments/execute/network_throughput/net.c
 ** Sample Execution Command:
-** gcc net.c -o net
-** ./net 10 "http://90.130.70.73/1MB.zip"
+** make net
+** ./net 1 5MB
 */
 
 int NUM_TRIALS;
-char* URL;
+char* DOWNLOAD_FILE_SIZE;
```

Comment:
I can't find the main method in the changes. Error message needs to be changed corresponding to this  DOWNLOAD_FILE_SIZE

Reply:
Fixed.